ush/wave_outp_spec.sh

```diff
@@ -214,13 +175,7 @@
      fi
    fi
   else
-    set +x
-    echo ' '
-    echo '***************************************************************** '
-    echo "*** FATAL ERROR : OUTPUT DATA FILE FOR BOUY ${bouy} NOT FOUND *** "
-    echo '***************************************************************** '
-    echo ' '
-    set_trace
+    echo "ERROR: OUTPUT DATA FILE FOR BOUY ${buoy} NOT FOUND"
```

Comment:
Now I'm just getting nit-picky :-D
```suggestion
    echo "ERROR: OUTPUT DATA FILE FOR BUOY ${buoy} NOT FOUND"
```

Reply:
Should be `FATAL ERROR`, no?

Reply:
@aerorahul I don't think so, but I could be persuaded.  Since this error occurs in an `ush` script, I am letting the ex-script handle the `FATAL ERROR` messaging.  I'm OK with writing `FATAL ERROR` messages here, too, but we should be consistent in the approach (i.e. all errors that will result in an eventual `err_exit` should write `FATAL ERROR`).

Reply:
@KateFriedman-NOAA Fixed, and that is the last "bouy" in the repository!

Reply:
Hooray! Thanks for finding and fixing those misspelled buoy instances. :)